cfpb/cfgov-refresh
# Sort imports per dev standards

This PR adds an `isort --check-only --diff` call to our Travis script. This lets us lint our Python imports for compliance with PEP8 and [Django's import style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports).

As part of this I've included an isort configuration file that should enforce those styles of imports and I've run `isort --recursive cfgov` over the codebase to adjust our imports to match. 

https://github.com/cfpb/cfgov-refresh/commit/bb97cbb140eae9889107bbe248039611b4e22ed9 has the isort configuration if you'd like to look independent of the fixing-imports changeset.

## Additions

- `isort` configuration
- `isort --check-only --diff` to Travis so that imports will be linted and corrections suggested.

## Changes

- Import orders in all relevant python files to comply with isort rules in `.isort.cfg`

## Testing

1. `git checkout sort-imports`
2. `pip install -r requirements/test.txt`
3. `git checkout master`
4. `isort --check-only --diff --recursive cfgov`
5. Notice lots of and lots of failures
6. `git checkout sort-imports`
7. `isort --check-only --diff --recursive cfgov`
8. Notice no failures!

## Checklist
